src/main/scala/com/timushev/sbt/updates/UpdatesPlugin.scala

```diff
@@ -13,10 +13,11 @@ object UpdatesPlugin extends AutoPlugin {
   override val projectSettings = Seq(
     dependencyUpdatesReportFile := target.value / "dependency-updates.txt",
     dependencyUpdatesExclusions := DependencyFilter.fnToModuleFilter(_ => false),
+    dependencyUpdatesFilter := DependencyFilter.fnToModuleFilter(_ => true),
```

Comment:
It seems that I was wrong — probably some traces of my experiments were left by mistake in a code. In a clean environment there is indeed no implicit conversion.